vrc/loaders/cparser.cc

```diff
@@ -134,6 +196,67 @@ enum CXChildVisitResult visit_function_body(CXCursor c, CXCursor parent, Visitor
     return result;
 }
 
+void retrieve_annotations(CXCursor c, VisitorState *state, size_t node_i)
```

Comment:
This is duplicate of `visit_function_decl`. Let's make them both work the same way, with either `visit` or `functor_visit`.